core/types/transaction.go

```diff
@@ -332,13 +336,13 @@ func (tx *Transaction) String() string {
 	From:     %s
 	To:       %s
```

Comment:
The String method is for debugging. We should probably delete it entirely. In this PR, I just tried to make the zero value printable in some way that doesn't crash when fields are nil.